app/assets/javascripts/surveys.js

```diff
@@ -175,6 +175,111 @@
+            '^/guidance/step-up-to-social-work-information-for-applicants(?:/|$)',
+            '^/government/publications/register-of-licensed-sponsors-workers(?:/|$)',
+            '^/guidance/scaled-scores-at-key-stage-1(?:/|$)',
+            '^/government/publications/financial-sanctions-consolidated-list-of-targets(?:/|$)',
+            '^/government/publications/list-of-qualifying-recognised-overseas-pension-schemes-qrops(?:/|$)'
+          ]
+        }
+      },
+      {
+        identifier: 'coming_to_the_UK_content_Transformation',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('August 25, 2017').getTime(),
+        endTime: new Date('October 1, 2017 23:59:50').getTime(),
+        url: 'https://signup.take-part-in-research.service.gov.uk/?utm_campaign=Coming_to_the_UK_content_Transformation&utm_source=Group_and_Transform_Content&utm_medium=gov.uk&t=GDS&id=17',
+        templateArgs: {
+          title: 'Help improve GOV.UK',
+          surveyCta: 'Answer some questions about yourself to join the research community.',
+          surveyCtaPostscript: 'This link opens in a new tab.'
+        },
+        activeWhen: {
+          path: [
+            '^/government/publications/chapter-18-naturalisation-at-discretion-nationality-instructions(?:/|$)',
+            '^/government/publications/chapter-8-appendix-fm-family-members(?:/|$)',
+            '^/government/publications/visit-guidance(?:/|$)',
+            '^/government/publications/sponsorship-codes-of-practice-for-skilled-workers(?:/|$)',
+            '^/government/publications/direct-family-members-of-european-economic-area-eea-nationals(?:/|$)',
+            '^/government/publications/general-grounds-for-refusal-about-this-guidance(?:/|$)',
+            '^/government/publications/chapter-6-information-about-applications-for-british-citizenship-nationality-instructions(?:/|$)',
+            '^/government/publications/eea-family-permits-eun02/eea-family-permit-eun02(?:/|$)',
+            '^/government/collections/nationality-instructions-volume-1(?:/|$)',
+            '^/government/collections/entry-clearance-basics-entry-clearance-guidance(?:/|$)'
+          ]
+        }
+      },
+      {
+        identifier: 'becoming_a_childminder',
+        surveyType: 'url',
+        frequency: 6,
+        startTime: new Date('August 25, 2017').getTime(),
+        endTime: new Date('October 1, 2017 23:59:50').getTime(),
+        url: 'https://signup.take-part-in-research.service.gov.uk/?utm_campaign=Becoming_a_childminder&utm_source=Group_and_Transform_Content&utm_medium=gov.uk&t=GDS&id=18',
+        templateArgs: {
+          title: 'Help improve GOV.UK',
+          surveyCta: 'Answer some questions about yourself to join the research community.',
+          surveyCtaPostscript: 'This link opens in a new tab.'
+        },
+        activeWhen: {
+          path: [
+            '^/register-childminder-childcare-provider(?:/|$)',
```

Comment:
I think we can drop the other `/register-childminder-chidcare-provider/<whatever>` paths as this one will match any subpages, and we've been asked to run it on all the subpages, so we can get away with just the root.  It does no harm to have the other explicit paths in here, but we can save a few bytes this way.

Need to keep the northern-ireland one though as it's a separate page.